recording/lib/storage.ex

```diff
@@ -18,6 +17,7 @@ defmodule Membrane.RTC.Engine.Endpoint.Recording.Storage do
           filename: String.t()
         }
 
-  @callback get_sink(config :: sink_config()) :: struct()
-  @callback save_object(config :: object_config()) :: :ok | {:error, String.t()}
+  @callback get_sink(recording_config :: recording_config(), opts :: any()) :: struct()
+  @callback save_object(object_config :: object_config(), opts :: any()) ::
+              :ok | {:error, String.t()}
```

Comment:
As we discussed, it would be beneficial to include some docs for these callbacks -- like [these](https://github.com/jellyfish-dev/jellyfish/blob/f9ff4130f4b0a46ff69ba447503fc9c6071d45cb/lib/jellyfish/component.ex#L44-L61)